src/commands/retention-policies/update.js

```diff
@@ -20,6 +20,9 @@ class RetentionPoliciesUpdateCommand extends BoxCommand {
 		if (flags['retention-length']) {
 			updates.retention_length = parseInt(flags['retention-length'], 10);
 		}
+		if (flags['non-modifiable']) {
+			updates.retention_type = this.client.retentionPolicies.retentionTypes.NON_MODIFIABLE;
+		}
```

Comment:
Just to clairify - we cannot update the policy to set it to `modifiable` right? We can only set it `non-modifiable` once it was created?

Reply:
You can create retention policy with either `modifiable` or `non-modifiable` type,  but you cannot change a policy to `modifiable` if it has been created `non-modifiable`.